integrations/slack/src/handlers/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './search';
```

Comment:
Should we remove the algolia search code completely? Or add a comment so that it's clear that it's not used

Reply:
gooood question. I wonder if, in the future, we might want to include it as part of the response anyhow. For now we could remove it though since it is unused.